lib/Doctrine/Common/DataFixtures/Loader.php

```diff
@@ -47,7 +47,7 @@ class Loader
     /**
      * Array of ordered fixture object instances.
      *
-     * @psalm-var array<class-string<OrderedFixtureInterface>, OrderedFixtureInterface>|list<OrderedFixtureInterface>
+     * @psalm-var array<class-string<FixtureInterface>, FixtureInterface>|list<FixtureInterface>
```

Comment:
I think, for consistency, `$orderedFixtures` property and `getLoader()` method should have the same type (in this case maybe `int` as a key makes more sense than a `list` meaning that it is an ordered array).